Survey Checks/README.md

```diff
@@ -0,0 +1,32 @@
+# Survey checks
+
+Here we provide detailed instructions for preparing your electronic survey with the aim of adding survey checks and creating a real-time high-frequency check dashboard.
+
+1. [Prepare Survey forms](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/1-prepare-scto-forms.md)
+2. [Adapt Survey forms](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/2-adapt-scto-forms.md)
+3. [Set up HFC Dashboard](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/3-set-up-hfc-dashboard.md)
+4. [Complementary analysis](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/4-complementary-analysis-r.md)
+5. [Add survey completion checks](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/5-survey-completion-checks.md)
+
+
+We also outline some [best practices and commonly faced issues](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/best-practices-and-issues.md) in setting up these real time HFC dashboards.
+
+## Relevant Links
+### SurveyCTO Forms and preload datasets </b>  
+1. Without module completion
+  - [Survey Form Template](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/scto/Survey%20Form%20Template%20(without%20module%20completion).xlsx)
+  - [Preload dataset](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/scto/Preloaded%20Data%20Sample.xlsx)
+  - [SurveyCTO form test and fill](https://boruis.surveycto.com/collect/demo_survey?caseid=)
+
+2. With module completion
+- Survey Form Template
+- [Preload dataset](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/scto/Preloaded%20Data%20Sample%20(With%20Module%20Completion).xlsx)
+- [SurveyCTO form test and fill](https://boruis.surveycto.com/collect/demo_survey_module_completion?caseid= )
+
+### Dashboard Template </b>  
+1. [HFC Dashboard Template (Viewer-mode)](https://docs.google.com/spreadsheets/d/16S2GlDgdeSuzAJEeML8ieDOjKYK7QzfHlrkJE6AwdC4/edit?usp=sharing)
+2. [HFC Dashboard Template (Editor-mode)](https://docs.google.com/spreadsheets/d/1iji2n0nSpS6tE4vOp9EwTb_TdvT-KC2J7wIMqw_q22M/edit?usp=sharing)  
+
+### Complementary analysis </b>  
```

Comment:
this is what I'd recommend moving to a subfolder, and here simply describing what checks should be done in the dashboard directly vs through complementary analysis